src/checkout/client/checkoutClient.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import {RemoteDataState} from '@influxdata/clockface'
+import {Checkout, toBackend} from 'src/checkout/utils/checkout'
+
+export const privateAPI = (path: string) => `/api/v2private/${path}`
+
+export class CheckoutClient {
+  private basePath: string
+
+  constructor() {
+    this.basePath = privateAPI('checkout')
+  }
+
+  completePurchase = async (formValues: Checkout): Promise<RemoteDataState> => {
+    const success = await axios
+      .post(`${this.basePath}/complete_purchase`, toBackend(formValues))
```

Comment:
not certain if this can currently be swapped for an IDPE generated route or if this exists on quartz

Reply:
this is `/checkout` in the generated unity routes it's `postCheckout`